core/src/test/java/org/apache/calcite/test/MaterializedViewSubstitutionVisitorTest.java

```diff
@@ -1259,6 +1259,259 @@ protected final MaterializedViewFixture sql(String materialize,
     sql(mv, query).ok();
   }
 
+  /**
```

Comment:
`https://github.com/apache/calcite/pull/1688`
Can these unit tests be supported?